test/events/test.triggered.with-gitcommitid.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
where is this new file used ?

Reply:
The file is used in the newly added unit tests.